src/test/cli/osdmaptool/clobber.t

```diff
@@ -19,6 +19,8 @@
   created \d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}\.\d+ (re)
   modified \d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}\.\d+ (re)
   flags 
+  full_ratio 0
+  nearfull_ratio 0
```

Comment:
that's just the default osdmap value; the mon sets it to something better during mkfs, or from pgmap during upgrade